R/stat-boxplot.r

```diff
@@ -59,7 +59,9 @@ StatBoxplot <- ggproto("StatBoxplot", Stat,
   },
 
   setup_params = function(data, params) {
-    params$flipped_aes <- has_flipped_aes(data, params, main_is_orthogonal = TRUE, group_has_equal = TRUE)
+    params$flipped_aes <- has_flipped_aes(data, params, main_is_orthogonal = TRUE,
+                                          group_has_equal = TRUE,
+                                          main_is_optional = TRUE)
```

Comment:
Tidyverse style indenting?

Reply:
Isn't this in line with the style guide?

Reply:
If we follow the tidyverse style guide, I believe these likes should be:

``` r
code <- "
    params$flipped_aes <- has_flipped_aes(data, params, main_is_orthogonal = TRUE,
                                          group_has_equal = TRUE,
                                          main_is_optional = TRUE)
"

styler::style_text(code)
#> Warning: Could not use colored = TRUE, as the package prettycode is not
#> installed. Please install it if you want to see colored output or see `?
#> print.vertical` for more information.
#> 
#> params$flipped_aes <- has_flipped_aes(data, params,
#>   main_is_orthogonal = TRUE,
#>   group_has_equal = TRUE,
#>   main_is_optional = TRUE
#> )
```

<sup>Created on 2020-03-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

But, it seems the guide is unclear about this point (I guess the "bad" example here describes somehow related rule, but there seems no corresponding text):

https://style.tidyverse.org/syntax.html#long-lines

Reply:
I'd probably write:

```R
#> params$flipped_aes <- has_flipped_aes(
#>   data, params,
#>   main_is_orthogonal = TRUE,
#>   group_has_equal = TRUE,
#>   main_is_optional = TRUE
#> )